File: easybuy-manager-web/src/main/java/com/easybuy/manager/controller/BrandController.java
package com.easybuy.manager.controller;
import java.util.List;

import com.easybuy.pojo.PageResult;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import com.alibaba.dubbo.config.annotation.Reference;
import com.easybuy.pojo.TbBrand;
import com.easybuy.sellergoods.service.BrandService;
/**
 * 品牌controller
 * @author Administrator
 */
@RestController
@RequestMapping("/brand")
public class BrandController {
    @Reference
    private BrandService brandService;
    /**
     * 返回全部列表
     * @return
     */
    @RequestMapping("/findAll")
    public List<TbBrand> findAll(){
        System.out.println(123445);
        return brandService.findAll();
    }

    @RequestMapping("/findPage")
    public PageResult  findPage(int page,int  rows){
        return  brandService.findPage(page,rows);

    }


    @RequestMapping("/findOne")
    public TbBrand  findOne(int id){
        System.out.println("-----------------------"+id);
        return null;
    }

}

File: easybuy-manager-web/src/main/webapp/js/controller/brandController.js
   //品牌控制层
    app.controller('brandController' ,function($scope,$controller,brandService){
        $controller('baseController',{$scope:$scope});//继承
        //读取列表数据绑定到表单中
        $scope.findAll=function(){
            brandService.findAll().success(
                function(response){
                    $scope.list=response;
                }
            );
        }

        $scope.findPage=function (page,rows) {
            brandService.findPage(page,rows).success(
                function (response) {
                    $scope.list=response.rows;
                    $scope.paginationConf.totalItems=response.total;//更新总记录数
                }
            )
        }


        $scope.findOne=function (id) {
            brandService.findOne(id).success(
                function(response){
                    $scope.entity= response;
                }
            )
        }





});
